daemons/execd/execd_commands.c

```diff
@@ -901,6 +901,28 @@ action_complete(svc_action_t * action)
                     }
                 }
             }
+        } else if (pcmk__strcase_any_of(cmd->action, PCMK_ACTION_MONITOR, PCMK_ACTION_STATUS, NULL) && 
+		  (cmd->interval_ms > 0)) { 
```

Comment:
I think probes would have the same issue. Probes will expect "not running" but could find it pending instead. There could be an argument that a probe should return "running" for pending but I think following the same timeout logic makes sense.

Reply:
I haven't looked into Probe much.
I'll look into it a bit more.

Reply:
@kgaillot 

I checked it.

I think what you mean when you say that Probe can handle the same thing here is something like this, what do you think?

----
If the status becomes PENDING during Probe, it will not respond immediately, but will continue to monitor at regular intervals until a timeout occurs, and will respond to controld when it gets a status other than PENDING.
If the timeout has passed, it will return PENDING.

In terms of processing, I think it is probably closer to the already existing start/stop follow up monitor than this time.

----
If I'm not mistaken, I will include this Probe change when I resubmit this PR.


Best Regards,
Hideo Yamauchi.

Reply:
Hi @kgaillot 

All right.
Prepare the pending monitor fixes and the pending probe fixes and submit the PR again.

Many thanks,
Hideo Yamauchi.